SUMMARY.md

```diff
@@ -0,0 +1,34 @@
+## Table of Contents
+
+* [Connecting to Database](content/ContectingToDatabase/ConnectingToDatabase.q.md) ** - minutes**
+    > Learn how you can use a specific class or command to establish a connection between PHP and PostgreSQL.
+
+* [PHP Data Objects](content/PDOClasses/PDOClasses.q.md) ** - minutes**
+    > In this lesson, you will learn about PHP Data Objects (`PDO`) classes and how to execute queries using `PDO::query()`.
+
+* [Prepared Statement](content/PreparedStatement/PreparedStatement.q.md) ** - minutes**
+    > In this lesson, you will learn how to prevent SQL injection using `PDO::prepare()` with its parameters in PHP.
+
+* [Prepared Statement with execute()](content/UsingExecute/UsingExecute.q.md) ** - minutes**
+    > Discover how you can execute prepared statements using `PDOStatement::execute()` with binding methods such as `bindParam()` and `bindValue()`.
+
+* [Prepared Statement with fetch()](content/UsingFetch/UsingFetch.q.md) ** - minutes**
+    > Learn how you can use `PDOStatement::fetch()` to fetch data and return a specific row of data from a result set.
+
+* [Prepared Statement with fetchColumn()](content/UsingFetchColumn/UsingFetchColumn.q.md) ** - minutes**
+    > Learn how you can use `PDOStatement::fetchColumn()` to fetch a column from the next row of a result set.
+
+* [Prepared Statement with fetchAll()](content/UsingFetchAll/UsingFetchAll.q.md) ** -minutes**
+    > Discover how you can use `PDOStatement::fetchAll()` to return an array that holds all the rows in a result set.
```

Comment:
Much better if we do it in this order
- fetchAll (`Select * from table`)
- fetch (`Select * from table where column = value`)
- fetchColumn (`Select username from table`)

From a simple query up to complex or with an additional command.

Reply:
This noted. Let me go ahead rearrange the order for the mentioned lessons. Thanks.